api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1131,35 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: Subscription is a high level consumption unit.
+    properties:
+      application_id:
+        type: string
+        example: 'price-service'
+        description: The id of application that will consume from the subscription.
+      event_types:
```

Comment:
I would leave this at this time as singular. We can offer (and think about the details) of providing multi topic streaming at a later stage.